modules/openapi-generator/src/main/resources/powershell/configuration.mustache

```diff
@@ -373,3 +373,134 @@ function Get-{{apiNamePrefix}}UrlFromHostSetting {
 
     }
 }
+
+<#
+.SYNOPSIS
+Sets the configuration for http signing.
+.DESCRIPTION
+
+Sets the configuration for the HTTP signature security scheme.
+The HTTP signature security scheme is used to sign HTTP requests with a key
+which is in possession of the API client.
+An 'Authorization' header is calculated by creating a hash of select headers,
+and optionally the body of the HTTP request, then signing the hash value using
+a key. The 'Authorization' header is added to outbound HTTP requests.
+
+Ref: https://openapi-generator.tech
+
+.PARAMETER KeyId
+KeyId for http signing 
+
+.PARAMETER KeyFilePath
+KeyFilePath for http signing
+
+.PARAMETER KeyPassPhrase
+KeyPassPhrase, if the http signing key is protected
+
+.PARAMETER HttpSigningHeader
+HttpSigningHeader list of http headers used to calculate the signature, Accepted values are 
+"(request-target), (created), (expires), Host, Date, Digest". If no headers are specified then '(created)' sets as default.
```

Comment:
User should be allowed to set any HTTP header present in the request, plus the special HTTP header fields. For example, they should be able to include `Content-Type` in the HTTP signature.